projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -1,6 +1,6 @@
 import { standardUser } from '../sample-data/shared-users';
 import { apiUrl } from '../support/utils/login';
-import { login } from './auth-forms';
+import { login, register } from './auth-forms';
 import { generateMail, randomString } from './user';
 
 interface TestProduct {
```

Comment:
Start by reading issue and PR description

Reply:
As an author make sure that you provide as much useful information as you can in the PR description:
- link to the ticket
- description of changes
- video with feature demo
- instructions (if testing the change is not that straightforward)
- any extra context needed
- describe tradeoffs in your approach
- point to files that are first to look at

Reply:
These points could be a checklist in PR template

Reply:
It could also be a good idea to start by testing the feature before you start reading the changes.

You might often go back to testing when you notice in code some edge cases that are not handled or you have some doubts about error handling